### ISSUE TITLE
Scrutinizer: `@var callable` instead `@var Closure`

### ISSUE THREAD

**Issue by distantnative** [k-next/router#3]
The Scrutinizer report suggests to rather use `@var callable` instead of `@var Closure` in doc blocks, e.g. in `src/Router/Route.php` for the `$action` property. Thoughts on this?

**Comment by lukasbestle** [comment]
Makes sense as `closure` is more general. The route action might as well be a static class method in array syntax etc.
